README.md

```diff
@@ -64,11 +64,24 @@ make run
   - Crowdsec LAPI available on which host and port.
 - CrowdsecLapiKey
   - string
+  - default: ""
   - Crowdsec LAPI generated key for the bouncer : **must be unique by service**. 
+- CrowdsecLapiTLSInsecureVerify
```

Comment:
TLS in maj or not ?

Reply:
Yes, In golang acronyme should be kept in uppercase

Reply:
So use Maj for other keys